ultra/ultra/scenarios/common/distributions.py

```diff
@@ -295,57 +295,92 @@
+                    },
                 },
                 "has_turn": False,
-                "deadlock_optimization": False,
+                "deadlock_optimization": True,
             },
             "west-north": None,  # blocking
             "west-south": {
                 "vehicles": 100,
-                "distribution": behavior_distribution,
+                "distribution": {
+                    "default": 0.7,
+                    "aggressive": 0.29,
+                    "cautious": 0.01,
+                    "blocker": 0.00,
+                },
                 "start_end_on_different_lanes_probability": 0.0,
                 "begin_time_init": {
-                    "func": basic_begin_time_init_func,
-                    "params": {"probability": prob_heavy},
+                    "func": burst_begin_time_init_func,
+                    "params": {
+                        "vehicle_cluster_size": (1, 2),
+                        "time_between_cluster": (30, 60),
+                        "time_for_each_cluster": 5,
+                    },
                 },
                 "has_turn": False,
                 "deadlock_optimization": False,
             },  # blocking
             "east-west": {
                 "vehicles": 1000,
-                "distribution": behavior_distribution,
+                "distribution": {
+                    "default": 0.7,
+                    "aggressive": 0.30,
+                    "cautious": 0.00,
+                    "blocker": 0.00,
+                },
                 "start_end_on_different_lanes_probability": 0.0,
                 "begin_time_init": {
-                    "func": basic_begin_time_init_func,
-                    "params": {"probability": prob_heavy},
+                    "func": burst_begin_time_init_func,
+                    "params": {
+                        "vehicle_cluster_size": (1, 4),
+                        "time_between_cluster": (25, 35),
+                        "time_for_each_cluster": 3,
+                    },
                 },
                 "has_turn": False,
-                "deadlock_optimization": False,
+                "deadlock_optimization": True,
             },
             "east-north": None,  # blocking
             "east-south": {
                 "vehicles": 100,
                 "distribution": behavior_distribution,
                 "start_end_on_different_lanes_probability": 0.0,
                 "begin_time_init": {
-                    "func": basic_begin_time_init_func,
-                    "params": {"probability": prob_heavy},
+                    "func": burst_begin_time_init_func,
+                    "params": {
+                        "vehicle_cluster_size": (1, 2),
+                        "time_between_cluster": (60, 70),
+                        "time_for_each_cluster": 2,
+                    },
                 },
-                "has_turn": True,
+                "has_turn": False,
```

Comment:
Also should `has_turn` be `True` here too? Since the `east-south` route has a turn?

Reply:
Should be go now 👌